.gitignore

```diff
@@ -1,5 +1,4 @@
 /clover.xml
-/composer.lock
```

Comment:
the composer lock should not exist in the repository - please do not remove this line.

Reply:
The file `composer.lock` is required by the CI worklfow.